fiberi18n/i18n.go

```diff
@@ -63,42 +63,54 @@ func (c *Config) initLocalizerMap() {
 	if _, ok := localizerMap.Load(lang); !ok {
```

Comment:
why is a new map created and filled in the upper part and not the c.localizerMap used?